src/FileConventions/Library.fs

```diff
@@ -246,3 +245,93 @@ let WrapText (text: string) (maxCharsPerLine: int) : string =
         $"{Environment.NewLine}{Environment.NewLine}",
         wrappedParagraphs
     )
+
+let DetectInconsistentVersion
```

Comment:
@realmarv 3 things to address about this function:

- Please rename to DetectInconsistentVersions.
- Please make it private or internal.
- Stop using `mutable` keyword.

Reply:
Done